packages/react-jsx-babel-plugin/__tests__/transform-jsx-to-react-createElement-test.js

```diff
@@ -0,0 +1,355 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+/* eslint-disable quotes */
+'use strict';
+
+const babel = require('@babel/core');
+const codeFrame = require('@babel/code-frame');
+
+function transform(input, options) {
+  return babel.transform(input, {
```

Comment:
Check out how `FreshBabelPlugin` test does `let {wrap} = require('jest-snapshot-serializer-raw');`. This will make snapshots a lot more readable.